README.md

```diff
@@ -33,6 +33,14 @@ pipenv shell
 pipenv install
 ```
 
+```
+NOTE:
+If you are using Windows 10 and you have a fresh Python install, you need to set your environment paths.
+To do this, open Powershell and type $env:PATH += ";yourpath" 
+This command appends a semicolon + your path to the system path file.
+You can verify this worked by typing $env:PATH
+```
```

Comment:
Please correct me if I'm wrong, but I'm not sure if this persists the PATH change across reboots. Have you tested this?

Reply:
I was under the impression that on windows you need to edit the environment variables in this UI for it to persist:
![image](https://user-images.githubusercontent.com/39167399/95346757-2db0c280-088a-11eb-833b-5446efdd5336.png)

Reply:
So was I…